default.env

```diff
@@ -84,4 +84,17 @@ export E2E_PAC_GITHUB_APP_ID=''
 # A github private key in base64 format used to set up Pac integerating with Github App
 # Note: How to get private key https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/managing-private-keys-for-github-apps
 # Required: only for running tests that are using PaC (see README.md for more details)
-export E2E_PAC_GITHUB_APP_PRIVATE_KEY=''
\ No newline at end of file
+export E2E_PAC_GITHUB_APP_PRIVATE_KEY=''
+
+# Only for upgrade tests
+# Branch with changes for upgrade (eg. new image tag...)
+# Example: quality-dashboard
+# Required: yes (for upgrade tests)
+export UPGRADE_BRANCH=quality-dashboard
+
+# Only for upgrade tests
+# Branch with changes for upgrade (eg. new image tag...)
+# Example: redhat-appstudio-qe
+# Required: no
+# Default value(if not specified): redhat-appstudio
+export UPGRADE_FORK_ORGANIZATION=redhat-appstudio-qe
```

Comment:
is there any default value specified? according to [this line](https://github.com/redhat-appstudio/e2e-tests/pull/652/files#diff-d7a1577b2d546e34aaf816bdd1b83edef28a66a555f049767bafa83268b7efbbR897) the default value is empty

Reply:
I have added the default value, but its more for make the code more readable, because in [here](https://github.com/redhat-appstudio/e2e-tests/pull/652/files#diff-8d3f02a956e3c919ac5ea21a43aa554db0d763ffa49bac99f9f294d88e612d59R341) it is using the origin for cloned infra-deployments, which is redhat-appstudio.  Thank you for this note!